src/pages/guides/gateway/architecture.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Adobe Graph Architecture Overview
+description: A description of the architecture and how requests are processed inside Adobe Graph.
+---
+
+# Architecture overview
+
+Depending on your role and use case for Adobe Graph, there are different ways to view the gateway architecture.
```

Comment:
We should agree on the preferred final naming for our service and its components and be consistent. If we call it Adobe Graph, we need to call it the same on the diagram. It would be great if we compile a glossary (in the internal or external docs), can you do that?

Reply:
"Adobe Graph" is the placeholder name we are using in the docs until a name is decided on. We don't currently have plans for a glossary, but it might be something we need for GA. I'll make a note.